src/appComponents/ErrorFallback.tsx

```diff
@@ -0,0 +1,193 @@
+  commentHelper?: string;
+  thanksMessage?: string;
+  sendButtonLabel?: string;
+  errorDetailsLabel?: string;
+}
+
+/**
+ * 
+ * @returns A form to submit user feedback.
+ * 
+ * @example
+const ErrorBoundary: FC<{ children?: ReactNode }> = ({ children }) => {
+  const { t: tFallback } = useTranslation('translations', {
+    keyPrefix: 'ERROR_BOUNDARY.FALLBACK',
+  });
+  return (
+    <Sentry.ErrorBoundary
+      // eslint-disable-next-line react/no-unstable-nested-components
+      fallback={({ error, componentStack, eventId }) => (
+        <ErrorFallback
+          error={error}
+          componentStack={componentStack}
+          eventId={eventId}
+          captureUserFeedback={Sentry.captureUserFeedback}
+          title={tFallback('MESSAGE_TITLE')}
+          formTitle={tFallback('MESSAGE_FEEDBACK')}
+          nameLabel={tFallback('NAME_LABEL')}
+          nameHelper={tFallback('NAME_HELPER')}
+          emailLabel={tFallback('EMAIL_LABEL')}
+          emailHelper={tFallback('EMAIL_HELPER')}
+          commentLabel={tFallback('COMMENT_LABEL')}
+          commentHelper={tFallback('COMMENT_HELPER')}
+          thanksMessage={tFallback('THANKS_FOR_FEEDBACK')}
+          sendButtonLabel={tFallback('SEND')}
+          errorDetailsLabel={tFallback('ERROR_DETAILS')}
+        />
+      )}
+    >
+      {children}
+    </Sentry.ErrorBoundary>
+  );
+};
+ */
+const ErrorFallback: FC<ErrorFallbackProps> = ({
+  error,
+  componentStack,
+  eventId,
+  captureUserFeedback,
+  title = TITLE,
+  formTitle = FORM_TITLE,
+  nameLabel = NAME_LABEL,
+  nameHelper = NAME_HELPER,
+  emailLabel = EMAIL_LABEL,
+  emailHelper = EMAIL_HELPER,
+  commentLabel = COMMENT_LABEL,
+  commentHelper = COMMENT_HELPER,
+  thanksMessage = THANKS_FOR_FEEDBACK,
+  sendButtonLabel = SEND,
+  errorDetailsLabel = ERROR_DETAILS,
+}) => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [comment, setComment] = useState('');
+  const [feedbackGiven, setFeedbackGiven] = useState(false);
+
+  const theme = useTheme();
+
+  const sendUserFeedback = (): void => {
+    const userFeedback = {
+      event_id: eventId,
+      name,
+      email,
+      comments: comment,
+    };
+    captureUserFeedback(userFeedback);
+    setFeedbackGiven(true);
+  };
+  return (
+    <Container>
+      <Paper variant='outlined' sx={{ p: 4 }}>
+        <Stack direction='column' spacing={1}>
+          <Stack direction='row' spacing={2} alignItems='center'>
+            <WarningIcon sx={{ color: theme.palette.error.main }} />
+            <Typography
+              variant='h2'
+              color={theme.palette.error.dark}
+              fontSize='1.8rem'
+            >
+              {title}
+            </Typography>
+          </Stack>
+          {feedbackGiven ? (
+            <Alert severity='success'>{thanksMessage}</Alert>
+          ) : (
+            <Box id='user-feedback'>
+              <Stack direction='column' spacing={1} maxWidth='82rem'>
```

Comment:
Where does that max-width come from ? Seems a bit arbitrary no ?

Reply:
That's true, it's completely arbitrary. It was because on very large screen, the form looks way too wide. Do you have any suggestion?